c/flake.nix

```
@@ -0,0 +1,42 @@
+{
+  description = "C Template";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs";
+    systems.url = "github:nix-systems/x86_64-linux";
+    flake-utils = {
+      url = "github:numtide/flake-utils";
+      inputs.systems.follows = "systems";
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils, systems }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+        pname = "hello-world"; #package name
+        version = "0.0.1";
+        src = ./.;
+        buildInputs = with pkgs; [
+          gcc
+        ];
+        nativeBuildInputs = with pkgs; [ ];
```

I guess this is what also many people need:

```suggestion
        nativeBuildInputs = with pkgs; [
          # pkg-config
        ];
```

---

Yep